packages/fxa-payments-server/src/routes/Subscriptions/index.tsx

```diff
@@ -71,6 +66,27 @@ export const Subscriptions = ({
 
   return (
     <div>
+      {updatePaymentStatus.loading &&
```

Comment:
Turns out React wasn't happy with these AlertBar changes happening inside the PaumentUpdateForm with its injectStripe wrapper. Not entirely sure why, but moving these up a level made a pile of errors go away.